### ISSUE TITLE
How to handle "Additional" tab

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Capabilities#223]
This happens with quite a few plugins, but this is WooCommerce.

We have these capabilities covered here:
<img width="1051" alt="Screen Shot 2021-10-28 at 11 18 38 AM" src="https://user-images.githubusercontent.com/3868028/139286054-d25eb0ac-d082-4f4a-99d8-8972c26254db.png">

We also have them duplicated under the "Additional" tab
<img width="599" alt="Screen Shot 2021-10-28 at 11 18 01 AM" src="https://user-images.githubusercontent.com/3868028/139286083-6d75512b-5eea-454d-9dea-77ce368b0fa3.png">

**Comment by olatechpro** [comment]
Hi @agapetry @stevejburge, when i checked the Editing and Additional tab. They seems to be different capabilities.

For example, on the editing tab, there's 'edit_products' (plural) while the additional tab own is 'edit_product' (singular) although if we're retaining it due to the singular and plural difference, they should go to WooCommerce tab.

So, what should we do here, remove duplicate even though there's singular plural difference or simply move the ones in additional tab to WooCommerce tab where they belongs ?

**Comment by stevejburge** [comment]
@olatechpro Good catch. Let's keep both for now. I can see both are used on the WooCommerce repo:
https://github.com/woocommerce/woocommerce/search?q=edit_products
https://github.com/woocommerce/woocommerce/search?q=edit_product

**Comment by olatechpro** [comment]
Okay, I'll move to woocommerce tab instead.

**Comment by agapetry** [comment]
@stevejburge @olatechpro This needs more actual usage testing.  This code sample is the meta capability usage I mentioned, and I think the others might be too.  See how the code is checking for a specific product ID? 

https://github.com/publishpress/PublishPress-Capabilities/issues/223#issuecomment-957960418

Since meta capabilities are internally referenced labels used to trigger ID-specific checks, giving them checkboxes alongside  regular capabilities has no useful function and would only cause confusion and support requests.

The only question here is whether there is some configuration mode where Woo applies these as normal capabilities, but I don't see that in the code snippets you showed.
